src/main/java/tc/oc/pgm/api/match/MatchPhase.java

```diff
@@ -50,4 +51,9 @@ public boolean canTransitionTo(MatchPhase next) {
         throw new IllegalStateException("Unknown transition state for " + next);
     }
   }
+
+  @Override
+  public String toString() {
+    return this.name().toLowerCase(Locale.ROOT);
```

Comment:
This seems like a breaking change? Why the change?

Reply:
I checked all usages of `MatchPhase` and this is not breaking. I use this [here](https://github.com/TheMolkaPL/PGM-1/blob/de337077ef99a96c0fc4051bc6359061ad049d73/src/main/java/tc/oc/pgm/listeners/ServerPingDataListener.java#L53) to serialize `MatchPhase` (`starting` instead of `STARTING`, etc). I could do this in `ServerPingDataListener`, but I felt that doing this here is more universal.